File: GameLibrary/src/cc/lib/zombicide/ZObjective.java
package cc.lib.zombicide;

import java.util.ArrayList;
import java.util.List;

import cc.lib.utils.Reflector;

/**
 * Created by Chris Caron on 3/10/22.
 */
public class ZObjective extends Reflector<ZObjective> {

    static {
        addAllFields(ZObjective.class);
    }

    final List<Integer> objectives = new ArrayList<>();
    final List<Integer> found = new ArrayList<>();
}

File: GameLibrary/src/cc/lib/zombicide/ZObjective.kt
package cc.lib.zombicide

import cc.lib.utils.Reflector
import java.util.*

/**
 * Created by Chris Caron on 3/10/22.
 */
class ZObjective : Reflector<ZObjective>() {
    companion object {
        init {
            addAllFields(ZObjective::class.java)
        }
    }

    val objectives: MutableList<Int> = ArrayList()
    val found: MutableList<Int> = ArrayList()
}